lib/siwapp/templates/template.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Siwapp.Templates.Template do
+  use Ecto.Schema
+
+  import Ecto.Changeset
+
+  @fields [:name, :template, :models, :print_default, :email_default, :subject, :deleted_at]
+
+  schema "templates" do
+    field :name, :string
+    field :template, :string
+    field :models, :string
```

Comment:
remove the models field (I know it was in the ruby version), but there is no need of that